src/core/theme/index.ts

```diff
@@ -1,39 +1,36 @@
 import { FlattenSimpleInterpolation } from 'styled-components';
-import {
-  typography,
-  TypographyProp,
-  internalTypographyTokens,
-  internalTypographyTokensProp,
-  TypographyDesignTokens,
-} from './typography';
-import { typographyUtils, spacingUtils, colorsUtils } from './utils';
 import {
   colors,
   shadows,
   gradients,
   outlines,
   ColorDesignTokens,
 } from './colors';
+import {
+  suomifiDesignTokens,
+  RawDesignTokens,
+  TypographyDesignTokens,
+} from 'suomifi-design-tokens';
 import { spacing, SpacingProp, SpacingDesignTokens } from './spacing';
 import { zindexes } from './zindexes';
 import { transitions } from './transitions';
 import { radius } from './radius';
-export {
-  TypographyProp,
-  internalTypographyTokens,
-  internalTypographyTokensProp,
-  SpacingProp,
-};
+export { TypographyProp, SpacingProp };
 
 export type SuomifiTokens = typeof importedTokens;
 export type DefaultSuomifiTokens = typeof defaultTokens;
 export type ColorProp = keyof typeof importedTokens.colors;
 export type SuomifiTheme = ReturnType<typeof suomifiTheme>;
 
+type TypographyProp = keyof typeof suomifiDesignTokens.typography;
+
+export { TypographyDesignTokens } from 'suomifi-design-tokens';
+
 export interface PartialTokens {
   colors?: Partial<ColorDesignTokens>;
   spacing?: Partial<SpacingDesignTokens>;
   typography?: Partial<TypographyDesignTokens>;
+  values?: Partial<RawDesignTokens>;
```

Comment:
Yes, I was thinking about possible ways for implementing this. The upside for not adding values to partial tokens as a way of customisation would be avoiding the risk of mismatch between values and css strings. The downside would be cases where you need to / want to use customised values for only part of the css strings, e.g. font size. Not sure if we need to allow this since it's only internally available in the library.

If partial custom values need to be used we could also define custom tokens or even add new tokens to design-system-tokens if valid for the use case.

=> Will simplify this and remove the raw tokens from partial tokens.